deepfence_worker/cronjobs/neo4j.go

```diff
@@ -100,7 +100,8 @@ func CleanUpDB(msg *message.Message) error {
 	if _, err = session.Run(`
 		MATCH (n:Node)
 		WHERE n.updated_at < TIMESTAMP()-$time_ms
-		AND NOT n.node_id IN ["in-the-internet", "out-the-internet"]
+		AND NOT n.node_id IN ["in-the-internet", "out-the-internet"] 
+		AND n.agent_running=true
```

Comment:
`agent_running` is deprecated, it is not used anywhere.
Its replacement is `active`.
We should remove this entirely from the agent itself